lib/redis/namespace.rb

```diff
@@ -243,8 +244,11 @@ class Namespace
     # This is used by the Redis gem to determine whether or not to display that deprecation message.
     @sadd_returns_boolean = true
 
+    # This is used by the Redis gem to determine whether or not to display that deprecation message.
+    @srem_returns_boolean = true
+
     class << self
-      attr_accessor :sadd_returns_boolean
+      attr_accessor :sadd_returns_boolean, :srem_returns_boolean
```

Comment:
I've added the boolean here so the use of `sadd_returns_boolean` in flipper-redis can be updated 
a new method `redis_srem_returns_boolean?` can be added similar to [redis_sadd_returns_boolean?](https://github.com/jnunemaker/flipper/blob/main/lib/flipper/adapters/redis.rb#L132-L134).
Then `redis_srem_returns_boolean?` could be used instead of `redis_sadd_returns_boolean?` 
which would make more sense in [this context]